src/main/java/seedu/souschef/storage/healthplan/XmlHealthPlanStorage.java

```diff
@@ -48,7 +48,7 @@ public XmlHealthPlanStorage(Path filePath) {
             logger.info("AppContent file " + filePath + " not found");
             return Optional.empty();
         }
-        logger.info("enters XmlHealthPlanStorage from read");
```

Comment:
I guess this change is related to storage?! Just make sure it's not mistakenly deleted.